projects/packages/analyzer/src/api/class-data-store.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+class Model {
```

Comment:
If the class is Model, should the file be named "class-model.php" instead of "class-data-store.php"?